src/ingest/IngestSpec.h

```diff
@@ -169,8 +178,10 @@ class IngestSpec : public nebula::common::Identifiable {
   std::string domain_;
   size_t size_;
   SpecState state_;
-  // macro date value in unix time stamp
-  size_t mdate_;
+  // list of macros (date, hour, minute, second) works with
+  std::vector<std::string> macros;
```

Comment:
this is not used and not complied with member naming convention. 

BTW, is this PR ready for review? Would love to wait for it's ready, so we can skip these small things if it's still in draft as I noticed in the PR labels.

Reply:
fixed, still need to add tests.